nyxtools/handlers.py

```diff
@@ -0,0 +1,38 @@
+import logging
+import pathlib
+
+from fabio import cbfimage
+from ophyd.area_detector_handlers import HandlerBase
+
+logger = logging.getLogger(__name__)
+
+
+class PilatusHandlerMX(HandlerBase):
+    spec = "AD_PILATUS_MX"
+
+    def __init__(self, fpath, seq_id):
+        self._seq_id = seq_id
+        self._fpath = pathlib.Path(f"{fpath}_{seq_id}.cbf").absolute()
+        if not self._fpath.is_file():
+            raise RuntimeError(f"File {self._fpath} does not exist")
+
+    def __call__(self, data_key="data", **kwargs):
```

Comment:
Why are you taking `**kwargs` if they are not used or passed on?

Reply:
I am addressing these issues now, thank you